### ISSUE TITLE
TypeError: Cannot read property 'content' of undefined

### ISSUE THREAD

**Issue by mauriciovigolo** [mauriciovigolo/file-matcher#1]
If there aren't file filters it throws the following error: 
```
TypeError: Cannot read property 'content' of undefined
    at FileMatcher.init (.../node_modules/file-matcher/lib/filematcher.js:188:28)
    at FileMatcher.find (.../node_modules/file-matcher/lib/filematcher.js:140:14)
```

**Comment by mauriciovigolo** [comment]
Error in documentation. The FindOptions interface was changed and the doc was outdated. 
Besides this the next version will fix that exception for a friendly message.

**Comment by mauriciovigolo** [comment]
Fix for this issue and documentation improvement released in v1.1.0.
